content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,77 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
 weight: 300
 toc: true
 ---
+
+### Settings Registry
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
+Many of the content processing rules that **Asset Processor** uses for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.
 
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
+{{< todo issue="https://github.com/o3de/o3de.org/issues/706" >}}
+To add project-specific or user-specific overrides, you can use the same pattern you would follow with other settings registry overrides. The linked ticket covers documenting the setttings registry override functionality.

```

Comment:
nit: we have issues not tickets in the GitHub side of things ie "The linked GitHub issue..."

Reply:
Fixed